src/main/scala/org/ergoplatform/modifiers/history/popow/NipopowProof.scala

```diff
@@ -86,6 +86,14 @@ case class NipopowProof(popowAlgos: NipopowAlgos,
       case (prev, next) => next.parentId == prev.id
     })
   }
+
+  /**
+   * Checks the interlink proofs of the blocks in the proof.
+   */
+  def hasValidProofs: Boolean = {
```

Comment:
It is not used. 

I tried to include it: 

```
def isValid: Boolean = {
    this.hasValidConnections && this.hasValidHeights && this.hasValidProofs
  }
```

but then PoPowAlgosSpec tests are failing